diff --git a/types/cypress-expect.d.ts b/types/cypress-expect.d.ts
index bea4c7f566d362f988b534ac4ef2791f4052ed6f..462909a9cf3d21fb8940e0b4df9293600a92b470 100644
--- a/types/cypress-expect.d.ts
+++ b/types/cypress-expect.d.ts
@@ -1,3 +1,2 @@
 // Cypress adds chai expect and assert to global
-declare var expect: Chai.ExpectStatic
 declare var assert: Chai.AssertStatic
diff --git a/types/index.d.ts b/types/index.d.ts
index 5167a954f11b866342c2a99dc4b7e301e7c7e12d..7b39abf9dc8c659baf4840f735df1adbdf0e396f 100644
--- a/types/index.d.ts
+++ b/types/index.d.ts
@@ -14,7 +14,6 @@
 /// <reference path="./lodash/index.d.ts" />
 /// <reference path="./sinon/index.d.ts" />
 /// <reference path="./sinon-chai/index.d.ts" />
-/// <reference path="./mocha/index.d.ts" />
 /// <reference path="./jquery/index.d.ts" />
 /// <reference path="./chai-jquery/index.d.ts" />
 
